Content.Client/VendingMachines/UI/VendingMachineItem.xaml

```diff
@@ -0,0 +1,25 @@
+<PanelContainer xmlns="https://spacestation14.io"
```

Comment:
shouldn't this just be a button rather than a panel container pretending to be a button? You can still add child controls to it anyways.

Reply:
Made it to be just a ``BoxContainer``.